hasura/migrations/1607444069515_init/up.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE "public"."User"("email" text NOT NULL, "agencyId" uuid NOT NULL, PRIMARY KEY ("email") );
```

Comment:
Is it OK to have this code in both the migration and the seed?

Reply:
No, this is a copy paste mistake